colorCode.py

```diff
@@ -0,0 +1,26 @@
+class ColorCode:
+    
+    def __init__(self):
+        self.MAJOR_COLORS = ['White', 'Red', 'Black', 'Yellow', 'Violet']
+        self.MINOR_COLORS = ["Blue", "Orange", "Green", "Brown", "Slate"]
+
+    def get_color_from_pair_number(self, pair_number):
```

Comment:
These methods seem like utility methods than class methods. Would it be useful to keep this out of ColorCode class? Please think over this.

Reply:
Hi @prithamvshetty , I am keeping this method inside the class, as now I have a different test_colorCode.py script.
I am creating the object instance only once in main.py and this method (get_color_from_pair_number) can be called from anywhere needed.
Keeping them outside will also work, but I generally like to keep them inside a class, easier to call the function with the object of the class defined. Do let me know if the change is still needed, or else we can resolve the comment :)

Reply:
No more comments 👍

Reply:
Thankyou @prithamvshetty  for the update :)